backend/.env.example

```diff
@@ -3,33 +3,31 @@ DEBUG=True
 APP_NAME=Fusion
 APP_VERSION=0.0.0
 
-SECRET_KEY=key
+UPLOADS_PATH=../uploads
+TEMPLATES_PATH=src/emails/templates
+
+ALGORITHM=HS256
+SECRET_KEY=KEY
+ACCESS_TOKEN_EXPIRE_MINUTES=43200  # 30 days
+
+VERIFICATION_CODE_MIN=100_000
+VERIFICATION_CODE_MAX=999_999
```

Comment:
is it working correct?

Reply:
maybe use
```suggestion
VERIFICATION_CODE_MIN=100000
VERIFICATION_CODE_MAX=999999
```

Reply:
it's working fine already, using underscores is recommended for better readability